tests/fail/intrinsics/float_to_int_32_inf1.stderr

```diff
@@ -1,8 +1,8 @@
-error: Undefined Behavior: `float_to_int_unchecked` intrinsic called on +Inf which cannot be represented in target type `i32`
+error: Undefined Behavior: `float_to_int_unchecked` intrinsic called on +Inff32 which cannot be represented in target type `i32`
```

Comment:
`Inff32` looks a bit strange... that's probably a bug in immediate printing. Can you file an issue to track this?

Reply:
Done: https://github.com/rust-lang/rust/issues/118221